ex03.ex

```diff
@@ -77,7 +81,12 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([head|tail],n) do
```

Comment:
## -4

Two problems here:

First, you are using `in`, which is a library function.

Second, the code seems very complex: if you're going to use `in` you could just write

~~~ elixir
def list_contains(list, n), do: n in list
~~~

However, I was looking for something like

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~